src/EditorFeatures/VisualBasicTest/Diagnostics/GenerateMethod/GenerateMethodTests.vb

```diff
@@ -301,7 +302,7 @@ Class C
 End Class")
         End Function
 
-        <WpfFact(Skip:="528229"), Trait(Traits.Feature, Traits.Features.CodeActionsGenerateMethod)>
+        <WpfFact(Skip:="528229")>
```

Comment:
Is this test still problematic?

Reply:
Thanks.  I got that working as well since i was poking around in this code.